e2e-tests/wait_for_hapi_start.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
Where is this file used? And is it possible to refactor the common functionality between here and `wait_for_start.sh`?